src/app/modules/customer-management/components/projects-type/components/create-project-type/create-project-type.component.ts

```diff
@@ -76,4 +80,9 @@ export class CreateProjectTypeComponent implements OnInit, OnDestroy {
   ngOnDestroy(): void {
     this.getCustomerIdSubscription.unsubscribe();
   }
+
+  onSearchChanges(searchValue: string): void {
```

Comment:
This function is identical to `onSearchChanges` of create-customer.ts
[Here](https://github.com/ioet/time-tracker-ui/blob/6d8b0b5003c99bb4f607f25c902af726e52b3c3c/src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.ts#L101)